ios/platform/Platform_iOS.mm

```diff
@@ -9,15 +9,17 @@ @implementation Platform
 - (instancetype)init {
     self = [super init];
     if (self) {
-        UIScreen *screen = [UIScreen mainScreen];
+        UIViewController *presentedViewController = RCTPresentedViewController();
```

Comment:
I'm not super familiar with the lower level of the RN framework, so there's maybe a better way to get the current view. I found this suggestion on Stack Overflow https://stackoverflow.com/a/57562922